source/include/ExecutionContext.h

```diff
@@ -94,6 +95,7 @@ class ExecutionContext
  public:
     ECONTEXT    Timer           _timer;           // TODO(PaulAustin): can be moved out of the execcontext once
                                                  // instruction can take injected parameters.
+    ECONTEXT    Boolean          _viPaused = false;
```

Comment:
This shouldn't be public. Make this private and add Get and Set public methods instead.